pDESy/model/base_project.py

```diff
@@ -612,6 +612,25 @@ def __allocate_component_to_workplace(
             ready_component = ready_task.target_component
             for workplace in ready_task.allocated_workplace_list:
                 if workplace.ID in target_workplace_id_list:
+                    if workplace.input_workplace_list != []:
+                        if (
+                            ready_component.placed_workplace != None
+                            and ready_component.placed_workplace != workplace
+                            and ready_component.placed_workplace not in workplace.input_workplace_list
+                        ):
+                            continue
+                        flag = False
```

Comment:
The word of `flag` should be changed for easy understanding.

Reply:
I'll change the flag name.
The initial value of the flag should have been set, but it seems to have been deleted after the test run. I'll fix it.

Reply:
I've changed the flag name, "cannnot_set_ready_child_component_to_workplace".
The line 622 is the line where the initial values are set.